combat-trainer.lic

```diff
@@ -1934,6 +1940,11 @@ class AbilityProcess
     echo 'Used Glyph of Mana!' if $debug_mode_ct
   end
 
+  # Barbarians without knowledge of the Power meditation won't
+  # see their activated abilities in the spell window.
+  # This means we can't always rely on DRSpells.active_spells
+  # to know if an ability is active or not for those characters.
+  # As a workaround, we monitor for the expiry messages.
```

Comment:
Documented _why_ Flags and expiry messages are used and not DRSpells.active_spells 😉